lib/src/global.dart

```diff
@@ -17,3 +17,72 @@ Future<void> installOfflineMapTiles(String tilesDb) async {
     },
   );
 }
+
+Future<List<OfflineRegion>> downloadListOfRegions({String accessToken}) async {
```

Comment:
I would suggest `loadListOfRegions`, or `getListOfRegions`. Since we aren't downloading anything over the network, "download" is a bit confusing.

Reply:
Hey, is there any other suggestions to PR? The name changing seems to be user friendly, but we would prefer to fix all other comments if it is possible.